dev/dune

```diff
@@ -34,6 +34,7 @@
     %{lib:coq.tactics:tactics.cma}
     %{lib:coq.vernac:vernac.cma}
     %{lib:coq.stm:stm.cma}
+    %{lib:coq.sysinit:sysinit.cma}
```

Comment:
The dev/db and dune_db* files should also be updated.

Reply:
I looked ad these files, but I don't know what to do (I never use Drop or ocamldebug...)